### DISCUSSION TITLE
How can I view the logs of a Dask Gateway worker?

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [nebari-dev/nebari#290] [Q&A] [answered]
How can I view the Dask Gateway worker logs when Dask dies thusly?
```
KilledWorker: ('xarray-longitude-064eca6a0765d769333bd8cb7eeaa885', <Worker 'tls://10.10.0.151:45321', name: 5d17b93e903f40f4a55c3d1b4e986f4d, memory: 0, processing: 1>)
```
I was hopeful that [the approach for Dask Kubernetes](https://kubernetes.dask.org/en/latest/api.html?highlight=logs#dask_kubernetes.HelmCluster.get_logs) would work, but sadly, no:
```
cluster.get_logs()
```
results in:
```
AttributeError: 'GatewayCluster' object has no attribute 'get_logs'
```
Is there a way to see what the logs show when a worker dies?

**Comment by rsignell-usgs** [comment, marked as answer]
I finally realized/remembered that if I open the Dask dashboard link in a new tab instead of in the JupyterLab extension, there is an "info" tab where I can look at the worker logs.

**Comment by Adam-D-Lewis** [comment]
I'm closing old discussions in a repo cleanup effort.  Feel free to re-open if needed.
